contracts/satoshi-bridge/src/kdf.rs

```diff
@@ -3,6 +3,29 @@ use crate::*;
 use crate::network::Address;
 use k256::elliptic_curve::sec1::ToEncodedPoint;
 
+#[near]
+impl Contract {
+    pub fn generate_public_key_pub(&self, path: String) -> String {
```

Comment:
A function useful for Bridge-SDK-RS. Overall, you can do without it, but having it makes things more convenient.

We need to get rid of the duplicated code in the function below and possibly replace path with tx_id or something like that.